src/SFA.DAS.EAS.Web/Orchestrators/EmployerCommitmentsOrchestrator.cs

```diff
@@ -861,7 +862,28 @@ public async Task<OrchestratorResponse<TransferFundedCohortsViewModel>> GetAllTr
             return await CheckUserAuthorization(async () =>
             {
                 var transferFundedCommitments = await GetAllCommitmentsOfStatus(accountId,
-                    RequestStatus.WithSenderForApproval, RequestStatus.RejectedBySender);
+                    RequestStatus.WithSenderForApproval, RequestStatus.RejectedBySender);//todo: remove rejected from here
```

Comment:
can we lose the todo now?

Reply:
it was already there, I wanted to check with you why it would already be there.